plugin/src/main/kotlin/org/jetbrains/intellij/versions/IdeaVersion.kt

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.intellij.versions
+
+@Suppress("ClassName")
+sealed class IdeaVersion(val value: String): java.io.Serializable {
+    object IDEA_IC_2017_2 : IdeaVersion(S_IDEA_IC_2017_2)
```

Comment:
Not quite sure all these pre-defined objects are needed here. There are too many different versions of IDEA, e.g. all IDEA releases include a set of minor versions (ideaIC:2017.2.1, 2.2, ...).